src/assets/scss/_settings.scss

```diff
@@ -493,6 +493,10 @@ $button-smaller-medium-shadow: 0 2px $button-background-hover;
 	}
 }
 
+@mixin input-focus() {
+	box-shadow: 0 0 0 0.25em rgba(174, 225, 92, 0.4); // $kiva-accent-green
+}
```

Comment:
What if we updated this to use the variable name? 
IE: background-color: rgba($kiva-accent-green, 0.4); 
That way when/if we update the $kiva-accent-green color, we don't have to hunt down this hard coded value and update it?

Reply:
Love it